util_file.py

```diff
@@ -71,9 +71,9 @@ def remove_timestring(filepath):
         return base_name
 
 
-def ls_remote(user, machine, directory, port=22):
+def ls_remote(user, machine, directory, port=22, custom_search=None):
```

Comment:
I can't quite follow where custom_search is used in this function?

Reply:
Its an error. Thanks.